dev/kyuubi-extension-spark-3-1/src/main/scala/org/apache/kyuubi/sql/sqlclassification/KyuubiGetSqlClassification.scala

```diff
@@ -35,29 +35,25 @@ import org.apache.kyuubi.sql.KyuubiSQLConf._
  *  If there have no this named jsonFile,
  *  the service will upload the default matching rule: sql-classification-default.json.
  */
-object KyuubiGetSqlClassification {
-
+object KyuubiGetSqlClassification extends Logging {
   private val jsonNode: Option[JsonNode] = {
-    SQLConf.get.getConf(SQL_CLASSIFICATION_ENABLED) match {
-      case true =>
-        val objectMapper = new ObjectMapper
-        var url: URL = Thread.currentThread().getContextClassLoader
-          .getResource("sql-classification.json")
-        if (url == null) {
-          url = Thread.currentThread().getContextClassLoader
-            .getResource("sql-classification-default.json")
-        }
-        val defaultSqlClassificationFile = url.getPath
-        Some(objectMapper.readTree(new File(defaultSqlClassificationFile)))
-      case false =>
-        None
+    if (SQLConf.get.getConf(SQL_CLASSIFICATION_ENABLED)) {
+      val objectMapper = new ObjectMapper
+      var url: URL = getClass.getClassLoader.getResource("sql-classification.json")
+      if (url == null) {
+        logInfo("sql-classification.json is not found, use default config instead")
+        url = getClass.getClassLoader.getResource("sql-classification-default.json")
```

Comment:
You provide an example `file:/Users/cathy/Desktop/tmp/spark-3.1.2-bin-hadoop2.7/jars/kyuubi-extension-spark-3-1_2.12-1.4.0-SNAPSHOT.jar!/sql-classification-default.json` if `sql-classification-default.json` is outside of the jar, but if `sql-classification-default.json` is inside a jar, what would the `url` look like? Does it also work?